src/main/java/teammates/common/util/Const.java

```diff
@@ -122,6 +122,7 @@ public static class ParamsNames {
         public static final String INSTRUCTOR_INSTITUTION = "instructorinstitution";
         public static final String IS_CREATING_ACCOUNT = "iscreatingaccount";
         public static final String IS_INSTRUCTOR = "isinstructor";
+        public static final String ACCOUNT_REQUEST_STATUS = "accountrequeststatus";
```

Comment:
```suggestion
        public static final String ACCOUNT_REQUEST_STATUS = "status";
```

No, wait. Actually, is it not possible to set the name itself as `status`, instead of `accountrequeststatus`? It's just that the variable name itself will still be `ACCOUNT_REQUEST_STATUS`, to prevent clashing with the other variable names in `ParamsNames`.